wiki/Tournaments/UKCC/10/en.md

```diff
@@ -263,6 +283,24 @@ See the [signups sheet](https://docs.google.com/spreadsheets/d/1HsZwZ-9NcBBMDksI
 
 ## Match results
 
+### Grand Finals
+
+#### Grand Final
+
+Saturday, 31 October 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Bubbleman** ![][flag_GB] | **7** | 1 | ![][flag_GB] majoreh | [#1](https://osu.ppy.sh/community/matches/93293452) |
+
+#### Loser Finals
```

Comment:
I can only find one instance of "Loser Finals" and that's in [UKCC/7](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/UKCC/7/en.md) (go figure), the rest use "Losers Finals"

```suggestion
#### Losers Finals
```